cells/std/lib/writeShellEntrypoint.nix

```diff
@@ -0,0 +1,212 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  l = nixpkgs.lib // builtins;
+  n2c = inputs.n2c.packages.nix2container;
+
+  stdImageDocs = "https://divnix.github.io/std/reference/std/lib/writeShellEntrypoint.html";
+  locales = nixpkgs.glibcLocales.override {allLocales = false;};
+
+  mkOCI = entrypoint: name: {
+    inherit name;
+    layers = [
+      (n2c.buildLayer {
+        deps = entrypoint.runtimeInputs;
+        maxLayers = 10;
+      })
+      # dependencies
+      (n2c.buildLayer {
+        deps = [entrypoint.package];
+        maxLayers = 90;
+      })
+    ];
+    contents =
+      []
+      ++ (l.optional (entrypoint.livenessProbe != null) entrypoint.livenessProbe)
+      ++ (l.optional (entrypoint.readinessProbe != null) entrypoint.readinessProbe);
+    config.Cmd = ["${entrypoint}/bin/entrypoint"];
+    config.Labels =
+      (l.optionalAttrs (entrypoint.package ? src) {
+        "org.opencontainers.image.source" = "file://${l.unsafeDiscardStringContext entrypoint.package.src}";
+      })
+      // (l.optionalAttrs (entrypoint.package.meta ? destination) {
+        "org.opencontainers.image.description" = entrypoint.package.meta.description;
+      })
+      // (l.optionalAttrs (entrypoint.package.meta ? license && entrypoint.package.meta.license ? spdxId) {
+        "org.opencontainers.image.licenses" = entrypoint.package.meta.license.spdxId;
+      })
+      // {
+        "org.opencontainers.image.url" = stdImageDocs;
+        "org.opencontainers.image.version" = l.getVersion entrypoint.package;
+        "org.opencontainers.image.title" = "Standard Image for ${l.getName entrypoint.package}";
+      };
+  };
+
+  mkDebugOCI = entrypoint: name: let
+    debug-banner = nixpkgs.runCommandNoCC "debug-banner" {} ''
+      ${nixpkgs.figlet}/bin/figlet -f banner "STD Debug" > $out
+    '';
+    debug-tools = with nixpkgs.pkgsStatic; [
+      bashInteractive.out
+      busybox
+    ];
+    debug-bin = nixpkgs.writeShellApplication {
```

Comment:
This here means we now have three versions of bash in the image... one is just `bash`, then `pkgsStatic.bash`, and `pkgsStatic.bashInteractive`.